onnx/backend/test/runner/__init__.py

```diff
@@ -199,8 +199,8 @@ def assert_similar_outputs(
                 f"and runtime type {type(outputs)} (known test={model_dir or '?'!r})"
             ) from e
         for i in range(len(outputs)):
-            if isinstance(outputs[i], (list, tuple)):
-                if not isinstance(ref_outputs[i], (list, tuple)):
+            if isinstance(outputs[i], list | tuple):
```

Comment:
As mentioned in https://github.com/onnx/onnx/pull/7219#issuecomment-3172676173, we should not update the isinstance calls